singularity-opac/photons/example_ascii/kap_plaw.txt

```diff
@@ -0,0 +1,19 @@
+ NRho 4 NT 4
+ rho_min 1.00000000e-14 rho_max 7.94328235e-01
```

Comment:
Are these inclusive bounds, e.g. the first opacity value is at rho=1e-14 and the last is at rho=7.94...e-01, rather than the opacity values being "bin centers" where their `rho = exp(log(rho_min) + (i + 0.5)*dlogrho)` for the `i`th opacity value?

Would it be worth adding a short paragraph to the `README` explaining this format?

Reply:
These are indeed inclusive bounds. Will add a description to the README.

Reply:
Just added a section to the README.md file.

Reply:
Looks great thanks!